requestmanager/asyncloader/responsecache/responsecache.go

```diff
@@ -67,13 +71,26 @@ func (rc *ResponseCache) AttemptLoad(requestID graphsync.RequestID, link ipld.Li
 
 // ProcessResponse processes incoming response data, adding unverified blocks,
 // and tracking link metadata from a remote peer
-func (rc *ResponseCache) ProcessResponse(responses map[graphsync.RequestID]metadata.Metadata,
+func (rc *ResponseCache) ProcessResponse(
+	ctx context.Context,
+	responses map[graphsync.RequestID]metadata.Metadata,
 	blks []blocks.Block) {
+
+	cids := make([]string, 0, len(blks))
+	for _, blk := range blks {
+		cids = append(cids, blk.Cid().String())
+	}
+	ctx, span := otel.Tracer("graphsync").Start(ctx, "cacheProcess", trace.WithAttributes(
+		attribute.StringSlice("blocks", cids),
```

Comment:
I'm not sure this is the best idea, it could be a long list of CIDs and that may not be helpful for tracing (maybe for logging, but maybe not for tracing). I could switch this to a `"blockCount"`.

Reply:
yea sometimes the cid lists can get long. we should just do block counts